extension/detect.js

```diff
@@ -0,0 +1,3 @@
+const isInstalledNode = document.createElement('div');
+isInstalledNode.id = 'apiary-browser-extension';
+document.body.appendChild(isInstalledNode);
```

Comment:
According to the docs:

> In the case of "document_start" (default value), the files are injected after any files from css, but before any other DOM is constructed or any other script is run. 

If I recall how the browser work, the javascript is executed *after* the _CSSOM_ is constructed - which in turn blocks the HTML rendering (to avoid style blinking).

Hence I would say the `body` element should always be there, ready to be manipulated.

For completeness, there's another (safer?) value:

> In the case of "document_end", the files are injected immediately after the DOM is complete, but before subresources like images and frames have loaded.

Anyway I am totally fine with changing the value with _document_end_